### PR TITLE
New statelesser

### PR DESCRIPTION
**Pull request by neutropolis**
Recovers the same behavior to automate instances that we reached here for `Field`s, but this time using the `P` & `Q` version and `LensAlg` as abstraction. Next, we'll introduce `TraversalAlg`s.

### CODE REVIEW

File: src/main/scala/package.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.hablapps
+
+import scalaz._
+import shapeless._
+
+package object statelesser extends LensAlgHom.Syntax {
+
+  type LensAlg[P[_], A] = LensAlgHom[MonadState, P, A]
```
Comment (jserranohidalgo) [inline src/main/scala/package.scala:8]: Better take this and its "companion" object to a new trait `LensAlgModule` (for instance), and place it in a different file. Then, you can import it through a mixin into the `statelesser` package.

**Review by jserranohidalgo — APPROVED** [review]
👍

File: src/main/scala/package.scala (hunk 1 above)
Comment (neutropolis) [reply src/main/scala/package.scala:8]: I was taking the [scalaz approach](https://github.com/scalaz/scalaz/blob/series/7.3.x/core/src/main/scala/scalaz/package.scala), where alias and companion object live in the package object. What is the benefit of placing them into an external module (if we take into account that the number of aliases won't be much larger) ?

Comment (jserranohidalgo) [reply src/main/scala/package.scala:8]: The question is: if Scala allowed us to place type aliases as top-level declarations, would you insist in placing them in the package object?

Comment (neutropolis) [reply src/main/scala/package.scala:8]: No, I wouldn't, but neither would I place them in a trait! :laughing: So, what is the point?

**Pull request merged**